service/src/test/java/edu/tamu/catalog/controller/PatronControllerTest.java

```diff
@@ -444,11 +444,11 @@ private void performPatronGetWithMockMVC(String url, String endpoint, PathParame
             Arguments.of(block, between(0, 1), once(), once(), respondJsonOk(blUserBadUUIDErrorPayload),
                 withStatus(BAD_REQUEST), "Bad%20UUID", status().isBadRequest()),
             Arguments.of(block, between(0, 1), once(), never(), respondJsonOk(blockEmptyCatalogPayload),
-                withNoContent(), USER_ID, status().isNotFound()),
+                withNoContent(), USER_ID, status().isInternalServerError()),
```

Comment:
Okay, so the way these tests work is that the `status().isInternalServerError()),` is the response the server ends.
By switching this from `isNotFound()` to `isInternalServerError()` is removing the 404 Not Found Test.

If the code does not properly handle the 404 and throws a 500, then there is something wrong with the code. (aka: the tests is properly failing that it did not bubble the 404.)
This should not be changed from a 404 to a 500.
The same for the other change below.

If we are transforming the 404 from FOLIO to a 500.

Reply:
I have a correction, the "emptyErrorPayload" is the response by the server and that is in the file blUserEmptyErrorPayload.json.

Reply:
We are only changing a 404 to a 500 for when the user response is found but the id is not present in the user.